plugins/k8sauditeks/falco.yaml

```diff
@@ -0,0 +1,14 @@
+plugins:
+  - name: k8sauditeks
+    library_path: /etc/falco/plugins/k8sauditeks/libk8sauditeks.so
+    init_config: '{"region": "us-east-1", "profile": "draios-dev-developer", "cluster_name": "RishZ", "polling_interval": 2}'
```

Comment:
Leftovers? I see no point in having a sample falco.yaml here anyway since we have all this info in the README

Reply:
:sweat_smile: